sdk/storage/azure-storage-blob/tests/test_blob_client.py

```diff
@@ -506,12 +506,10 @@ def test_user_agent_default(self, resource_group, location, storage_account, sto
 
         def callback(response):
             self.assertTrue('User-Agent' in response.http_request.headers)
-            self.assertEqual(
-                response.http_request.headers['User-Agent'],
-                "azsdk-python-storage-blob/{} Python/{} ({})".format(
+            assert ("azsdk-python-storage-blob/{} Python/{} ({})".format(
                     VERSION,
                     platform.python_version(),
-                    platform.platform()))
+                    platform.platform())) in response.http_request.headers['User-Agent']
```

Comment:
Changed this since the live run in CI adds a random build number in the end and the test fails because there is no exact match. similar in other tests too.,